django/cantusdb_project/main_app/forms.py

```diff
@@ -209,12 +209,18 @@ class Meta:
     century.widget.attrs.update({"class": "form-control custom-select custom-select-sm"})
 
     current_editors = forms.ModelMultipleChoiceField(
-        queryset=get_user_model().objects.all().order_by("last_name"), required=False
+        queryset=get_user_model().objects.filter(
```

Comment:
If we do `project_manager OR contributor OR Editor`, perhaps we don't need the filtering? That's just all users.

Reply:
Will there never come a time when a user is given an account and not assigned to any group?

Reply:
Also, I did the filtering in case additional groups may be added in the future. If these two cases aren't likely to happen, I'll update accordingly.